virtcontainers/network.go

```diff
@@ -63,6 +64,28 @@ func (n NetInterworkingModel) IsValid() bool {
 	return 0 <= int(n) && int(n) < int(NetXConnectInvalidModel)
 }
 
+func fileExists(path string) bool {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return false
+	}
+
+	return true
+}
+
+// haveVhostNetKernelModule returns true if the vhost_net module exists
+// (either loaded or available to be loaded)
+func haveVhostNetKernelModule() bool {
+	// First, check to see if the vhost module is already loaded
+	if fileExists("/sys/module/vhost_net") {
+		return true
```

Comment:
maybe we don't need func `fileExsits` here since it's really simple : just a `Stat` and if we really need this func , it should be a `utils` not just for `network` itself.

Reply:
ok